index.html

```diff
@@ -1059,6 +1059,11 @@ <h3 id="set-permission-command">
           The <a>remote end steps</a> are:
         </p>
         <ol>
+          <li>If the <a>top-level browsing context</a> is "no longer open", return <a>error</a>
```

Comment:
Missing links for some terms, e.g. "no longer open", "no such Window", "Handle any user prompts".
You can refer to step 3 and 4 in https://w3c.github.io/sensors/#section-extension-commands.

And should it be the [current top-level browsing context](https://w3c.github.io/webdriver/#dfn-current-top-level-browsing-context) instead? I'm not quite sure.